docs/high-availability/srv_kubernetes/apiserver.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
`srv_kubernetes/apiserver.key` and `srv_kubernetes/apiserver.crt` are not used. We should remove them.